generic-request-example/src/#generic_request/handler.ts

```diff
@@ -0,0 +1,45 @@
+import { OperationHandlerSetup } from '@trayio/cdk-dsl/connector/operation/OperationHandlerSetup';
+import { BizzaboCdkAuth } from '../BizzaboCdkAuth';
+import { GenericRequestInput } from './input';
+import { GenericRequestOutput, GetAccessTokenRes } from './output';
+import { OperationHandlerResult } from '@trayio/cdk-dsl/connector/operation/OperationHandler';
+
+// This operation is used to get customized OAuth2 access tokens
+export const genericRequestHandler =
+	OperationHandlerSetup.configureHandler<
+		BizzaboCdkAuth,
+		GenericRequestInput,
+		GenericRequestOutput
+	>((handler) =>
+		handler.usingHttp((http) =>
+			http
+				.post('https://auth.bizzabo.com/oauth/token')
+				.handleRequest((ctx, input, request) => {
+					return request
+						// Send body data according to the API authentication flow
+						.withBodyAsJson({
+							// Grab values from the authentication form "auth_form_input"
+							client_id: input.auth_form_input.client_id,
+							client_secret: input.auth_form_input.client_secret,
+							account_id: input.auth_form_input.account_id,
+							audience: 'https://api.bizzabo.com/api',
+							grant_type: 'client_credentials',
+						})
+				})
+				.handleResponse((ctx, input, response) => {
+					return response.parseWithBodyAsJson<GetAccessTokenRes>((body) => {
+						return OperationHandlerResult.success({
+							// You must respond with status_code, headers, and body
```

Comment:
why is this a must ?

Reply:
API service expect this otherwise it throws error